11.0/Dockerfile

```diff
@@ -0,0 +1,113 @@
+FROM debian:jessie
+MAINTAINER Camptocamp
+
+# create the working directory and a place to set the logs (if wanted)
+RUN mkdir -p /opt/odoo /var/log/odoo
+
+WORKDIR "/opt/odoo"
+
+COPY ./base_requirements.txt ./
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN set -x; \
+        sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+        && apt-get update \
+        && apt-get install -y --no-install-recommends \
+            antiword \
+            ca-certificates \
+            curl \
+            ghostscript \
+            graphviz \
+            less \
+            nano \
+            node-clean-css \
+            node-less \
+            poppler-utils \
+            python \
+            python-libxslt1 \
+            python-pip \
+            xfonts-75dpi \
+            xfonts-base \
+            # build packages to clean after the pip install
+            build-essential \
+            python-dev \
+            libfreetype6-dev \
+            libxml2-dev \
+            libxslt1-dev \
+            libsasl2-dev \
+            libldap2-dev \
+            libssl-dev \
+            libjpeg-dev \
+            zlib1g-dev \
+            libfreetype6-dev \
+            tcl expect \
+        && curl -s https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+        && apt-get update \
+        && apt-get install -y --no-install-recommends \
+            postgresql-client \
+            libpq-dev \
+        && curl -o wkhtmltox.deb -SL http://nightly.odoo.com/extra/wkhtmltox-0.12.1.2_linux-jessie-amd64.deb \
+        && echo '40e8b906de658a2221b15e4e8cd82565a47d7ee8 wkhtmltox.deb' | sha1sum -c - \
+        && dpkg --force-depends -i wkhtmltox.deb \
+        && apt-get -y install -f --no-install-recommends \
+        && pip install -U pip && pip install -r base_requirements.txt \
```

Comment:
Warning this might fail due to https://github.com/timotheus/ebaysdk-python/issues/182
The workaround is to set locals.

Reply:
By the way this was fixed with ebaysdk 2.1.5